src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
 
-**User journey** is used to describe a sequence of actions taken by either a real or simulated user.
+<dt id="scenario">Scenario</dt>
+<dd>A special option that models plausible events that an application could experience. To model high traffic, a scenario might have virtual users making concurrent requests over multiple script iterations. Configure scenario behavior with [executors](#executors)
+<p>Read more: [Scenarios reference](/using-k6/scenarios)</p></dd>
 
-### User Scenario
+<dt id="service-level-agreement">Service-level agreement</dt>
+<dd>*(Or SLA)*. An agreement between a service provider and another party, often a user of the service, promising that the availability of the service will meet a certain level during a certain period.</dd>
 
-**User Scenario** is a synonym for [user journey](#user-journey).
+<dt id="service-level-indicator">Service-level indicator</dt>
+<dd>*(Or SLI)*. A metric that measures whether a service meets its [service-level objective](#service-level-objective). In performance monitoring, and SLI could be the number of successful requests against the service during a specified period.</dd>
 
-### Vertical Scalability
+<dt id="service-level-objective">Service-level objective</dt>
+<dd>*(Or SLO)*. An actual target, either internal or part of the [service-level agreement](#service-level-agreement), for the availability of the service. This is often expressed as a percentage (99,2%, for instance). If the service meets or exceeds this target, it's within its "error budget"</dd>
 
-**Vertical scalability** is a trait describing to what degree a system under test’s performance or capacity may be increased by adding more hardware resources to a node (RAM, cores, bandwidth, etc.).
+<dt id="smoke-test">Smoke test</dt>
+<dd>A test that verifies whether the system under test can handle a minimal amount of load without any issues. Testers commonly use smoke tests to make sure that everything works as intended under optimal conditions. After the smoke test, they can advance to any of the other performance test types.
+<p>Read more: [Smoke Testing](/test-types/smoke-testing)</p></dd>
 
-### Virtual Users
+<dt id="soak-test">Soak test</dt>
+<dd>A test that tries to uncover performance and reliability issues stemming from a system being under pressure for an extended duration.
+<p>Read more: [Soak Testing](/test-types/soak-testing)</p></dd>
 
-**Virtual Users**, or **VUs** are used to perform separate and concurrent executions of your test script. They can make HTTP(s) and WebSocket requests against a webpage or API.
+<dt id="stability">Stability</dt>
+<dd>A system under test’s ability to withstand failures and errors.</dd>
 
-Virtual Users, although emulated by k6 itself, can be used to mimic the behavior of a real user.
+<dt id="stress-test">Stress test</dt>
```

Comment:
Stress test / Soak Test / Load Test / Smoke test --> must be in sync with the descriptions at https://k6.io/docs/test-types/

If not, consider removing it for now.